packages/assets-controllers/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [15.0.1]
+### Changed
+- Bump dependency on `@metamask/preferences-controller` to ^4.4.3
```

Comment:
Nit: Since this is also a peer dependency update, we can be a little more detailed here:

``` suggestion
- Bump dependency and peer dependency on `@metamask/preferences-controller` to ^4.4.3